src/Entity/ValuationRequest.php

```diff
@@ -0,0 +1,255 @@
+<?php
+
+namespace Jedkirby\ValPal\Entity;
+
+/**
+ * @codeCoverageIgnore
+ */
+class ValuationRequest
+{
+    /**
+     * @var string
+     */
+    private $buildingName;
+
+    /**
+     * @var string
+     */
+    private $subBuildingName;
+
+    /**
+     * @var string
+     */
+    private $number;
+
+    /**
+     * @var string
+     */
+    private $street;
+
+    /**
+     * @var string
+     */
+    private $dependentStreet;
+
+    /**
+     * @var string
+     */
+    private $postcode;
+
+    /**
+     * @var string
+     */
+    private $email;
+
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var string
+     */
+    private $phone;
+
+    /**
+     * @var string
+     */
+    private $reference;
+
+    /**
+     * @param string $buildingName
+     * @param string $subBuildingName
+     * @param string $number
+     * @param string $street
+     * @param string $dependentStreet
+     * @param string $postcode
+     * @param string $email
+     * @param string $name
+     * @param string $phone
+     * @param string $reference
+     */
+    public function __construct(
+        $buildingName,
```

Comment:
Would it be worth wrapping these values in a value object class and passing that instead of having _lots_ of params?